charts/kube-prometheus-stack/templates/exporters/kubelet/servicemonitor.yaml

```diff
@@ -95,7 +101,11 @@ spec:
     scrapeTimeout: {{ .Values.kubelet.serviceMonitor.scrapeTimeout }}
     {{- end }}
     honorLabels: {{ .Values.kubelet.serviceMonitor.honorLabels }}
+    {{ if .Values.kubelet.serviceMonitor.trackTimestampsStaleness }}
```

Comment:
Since `trackTimestampsStaleness` is not set for the probe endpoint, the condition feels pointless.

Reply:
Oops, that was supposed to be on resources.